package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "devDependencies": {
+    "danger": "^9.2.10",
+    "typescript": "^3.7.5"
```

Comment:
Notice that, on top of Danger, we have TypeScript as well. That's because [our `Dangerfile`s are written in TypeScript](https://github.com/Automattic/peril-settings/tree/5d22258ca46c3649f2cb863cfac2d16f12c4cba0), so Danger needs the TypeScript transpiler to be in the `node_modules/` folder in order to process them.